pkg/util/util.go

```diff
@@ -54,3 +54,15 @@ func ErrList(e []error) error {
 	}
 	return nil
 }
+
+func InterfaceSlicesEqual(a, b []interface{}) bool {
+	if len(a) != len(b) {
+		return false
+	}
+	for i, v := range a {
+		if v != b[i] {
+			return false
```

Comment:
Will the order of the status.storedVersions in the two objects be always same? Will this matter since we are comparing items at the same index